front-end/virtual_pet/lib/flame/game/vitural_pet_game.dart

```diff
@@ -1,12 +1,20 @@
 import 'package:flame/game.dart';
```

Comment:
Can we rename this file so it is `virtual_pet_game.dart` and not `vitural_pet_game.dart`?

Reply:
Really nice otherwise, very clean code 👍

Reply:
Good catch!